test_scripts/Smoke/API/046_OnSeekMediaClockTimer_PositiveCase_SUCCESS.lua

```diff
@@ -0,0 +1,66 @@
+---------------------------------------------------------------------------------------------------
+-- User story: Smoke
+-- Use case: OnSeekMediaClockTimer
+-- Item: Happy path
+--
+-- Requirement summary:
+-- [OnSeekMediaClockTimer]: getting SUCCESS:UI.SetMediaClockTimer()
+
```

Comment:
@HSavynetska Please add sections: 
1. splitter (`-------`)
2. Description
3. Pre-conditions
4. Steps
5. Expected

See other scripts in `Smoke/API`